lib/publiccloud/utils.pm

```diff
@@ -256,7 +256,8 @@ sub gcloud_install {
 }
 
 sub get_ssh_private_key_path {
-    return (is_azure()) ? '~/.ssh/id_rsa' : '~/.ssh/id_ed25519';
+    my $root_path = is_openstack ? '/root/' : '~/';
```

Comment:
Please don't create such complication and just expand the tilda where it's needed if it's even needed. I think that terraform can handle tilda.